index.html

```diff
@@ -1651,10 +1651,12 @@ <h3>Credential Subject</h3>
           <dt><dfn class="export"
                    id="defn-credentialSubject">credentialSubject</dfn></dt>
           <dd>
-The value of the <code>credentialSubject</code> <a>property</a> is defined as
-a set of objects that MUST contain one or more claims that are each related to a
-<a>subject</a> of the <a>verifiable credential</a>. Each object MAY contain an
-<code>id</code>, as described in Section <a href="#identifiers"></a>.
+The value of the <code>credentialSubject</code> <a>property</a> is defined as a
+set of objects where each object MUST be the <a>subject</a> of one or more <a>claims</a>,
+which MUST be serialized inside the <code>credentialSubject</code> <a>claim</a>.
```

Comment:
Yes, please do not conflate the term "claim" with "property". There is no "credentialSubject claim"; a claim is a subject->property->value triple. Using the word "property" here at the end is much better.

Reply:
Yep, that's great, thanks. Neither option is really precise, since a property is just a URL that doesn't contain anything, but I think either is clear enough.